### PR TITLE
Promisify sql queries

### PR DESCRIPTION
**Pull request by picasdan9**
- promisify pool.query and rewrite pool.query calls to async/await
- rewrite emailTransporter to async/await
- call parseInt directly when extracting the id in path parameter 
- change status condition in getPastOrdersForBuyer sql statement to >=4 (in progress)

### CODE REVIEW

File: code/handlers/itemHandler.js [hunk 1]
```diff
@@ -193,7 +205,9 @@ function handleDisableRequest (req, res) {
   let showQueryString = itemQueries.getItem(id)
 
   // Execute query strings
-  pool.query(showQueryString, function (err, rows, fields) {
+  try {
+    // Validate user rights and action
+    const rows = pool.query(showQueryString)
     if (!rows.length)
       return res.json({
         success: false,
```
Comment (elenacyl) [inline code/handlers/itemHandler.js:213]: Seems like rows are not being detected correctly

File: code/handlers/resetHandler.js [hunk 2]
```diff
@@ -12,27 +12,22 @@ const usePasswordToGenerateToken = ({
...
+  try {
+    const rows = await pool.query(queryUser)
+    if (!rows.length) 
       return res.json({
         success: false,
         message: "Invalid Email or Password"
       })
-    }
+    const user = rows[0]
     const token = usePasswordToGenerateToken(user)
```
Comment (elenacyl) [inline code/handlers/resetHandler.js:29]: Does this work on your end? Not sure what's the issue but it doesnt seem to work on my end.

Comment (picasdan9) [reply code/handlers/resetHandler.js:29]: @elenacyl i think it worked for me when i tested this. did you go to the reset page on localhost and not the actual app (which is the link sent by the email)?

Comment (elenacyl) [reply code/handlers/resetHandler.js:29]: Oh ok. My bad! No issues here then!

File: code/handlers/itemHandler.js (hunk 1 above)
Comment (picasdan9) [reply code/handlers/itemHandler.js:213]: I forgot to add the `await` keyword for the query before this line of code. Just added in the new commit and tested the endpoint. Sorry for the mistake!

**Pull request merged**